crates/starknet-server/tests/get_transaction_by_hash.rs

```diff
@@ -176,10 +167,8 @@ mod get_transaction_by_hash_integration_tests {
             .unwrap();
 
         if let starknet_rs_core::types::Transaction::DeployAccount(deploy) = result {
-            assert_eq!(
-                deploy.transaction_hash,
-                FieldElement::from_hex_be(DEPLOY_ACCOUNT_TRANSACTION_HASH).unwrap()
-            );
+            let expected = "0x02b4b37075273f836aa055c7a53e4e2635abcbd776ebef2ab1b74abd7235ac06";
```

Comment:
Can we use const at the beginning of tests like it was before?

Reply:
See: https://github.com/0xSpaceShard/starknet-devnet-rs/pull/238#discussion_r1389116033